.rhcicd/clowdapp-backend.yaml

```diff
@@ -166,6 +217,25 @@ objects:
                   secretKeyRef:
                     name: notifications-backend-db
                     key: ${DB_SECRET_PASSWORD_KEY}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: floorplan
+  data:
+    floorplan.yaml: |
+      - prefix: insights/notifications/splunk
+        query: >-
+          SELECT 
+            DISTINCT "account_id" 
+            FROM "endpoints"
+            WHERE "endpoint_type"=3 AND "endpoint_sub_type"='splunk';
+      - prefix: insights/notifications/accounts
+        query: >-
+          SELECT
+          "applications"."id", "applications"."name", COUNT("applications"."id") AS "applications_count" 
+          FROM "applications" 
+          INNER JOIN "endpoint_email_subscriptions" ON "applications"."id"="endpoint_email_subscriptions"."application_id" 
+          GROUP BY "applications"."id" HAVING COUNT("applications"."id") > 1;
```

Comment:
Is Postgres happy with this query? When a SQL aggregation function (`COUNT` here) is used, all unaggregated columns should be part of the `GROUP BY` clause. It may work in Postgres because `applications.name` is functionally dependent on `applications.id` though.

Reply:
Yes, Postgres is happy with it